docs/img/run_task_on_celery_executor.puml

```diff
@@ -0,0 +1,55 @@
+@startuml
```

Comment:
I think it would be great to add in a comment here how to generate new diagram in case someone changes it (I guess it is done via https://plantuml.com/  but I am guessing now.

Reply:
I added link to the instruction above.